llvm/lib/Transforms/Instrumentation/MemProfiler.cpp

```diff
@@ -166,6 +166,11 @@ static cl::opt<bool>
                                      "context in this module's profiles"),
                             cl::Hidden, cl::init(false));
 
+static cl::opt<std::string>
+    MemprofDefaultOptions("memprof-default-options",
```

Comment:
Perhaps "memprof-runtime-default-options" to make it clear that it affects the compiler-rt runtime?